src/resources/NotFoundSite/NotFoundSite.module.css

```diff
@@ -0,0 +1,47 @@
+.notFoundSiteWrapper {
+  margin-left: 20px;
+  margin-right: 20px;
+}
+
+.errorMessage {
+  font-size: 14px;
+  color: darkslategray;   
+}
+
+.contentText {
+  font-size: 20px;
+}
+
+.flexContainer {
+  display: flex;
+}
+
+.leftContainer {
+  flex: 3;
+}
+
+.rightContainer {
+  flex: 1;
```

Comment:
Another nitpick:
The page looks more symmetrical if this line of styling is skipped

Reply:
I agree